.github/CODEOWNERS

```diff
@@ -100,6 +100,9 @@
 /.werft/*installer-tests* @gitpod-io/engineering-self-hosted
 /.werft/jobs/build/self-hosted-* @gitpod-io/engineering-self-hosted
 
+/preview/infrastructure/harvester @gitpod-io/platform
```

Comment:
We can remove these now that the files have been moved as we already own /dev/preview ☺️

Reply:
I've updated them, but I would like to keep them explicit, as we would like to separate them in the future.